docs/api/geo_datasets.csv

```diff
@@ -14,6 +14,7 @@ Dataset,Type,Source,Size (px),Resolution (m)
 `iNaturalist`_,Points,Citizen Scientists,-,-
 `LandCover.ai Geo`_,"Imagery, Masks",Aerial,"4,200--9,500",0.25--0.5
 `Landsat`_,Imagery,Landsat,"8,900x8,900",30
+`L8Biome`_,"Imagery, Masks",Landsat,"8,900x8,900",30
```

Comment:
```suggestion
`L8Biome`_,"Imagery, Masks",Landsat,"8,900x8,900","15, 30"
```
Technically there are both 15 m and 30 m resolution bands. We should probably be more specific in the other datasets too, but we can do that in a different PR.